common-healing.lic

```diff
@@ -203,16 +203,14 @@ module DRCH
   def bind_wound(part, person = 'my')
     snap = [DRC.left_hand, DRC.right_hand]
     if 'You skillfully remove' == DRC.bput("tend #{person} #{part}", 'You work carefully at tending', 'That area has already been tended to', 'That area is not bleeding', 'You fumble', 'You \w+ remove', 'You (carelessly|foolishly)', 'You deftly remove')
-      waitrt?
```

Comment:
Dealing with RT is one of the best things about bput. I don't see any reason to clutter this up with the extra call.